bedrock/firefox/firefox_details.py

```diff
@@ -209,6 +209,9 @@ def get_download_url(self, channel, version, platform, locale,
         _locale = 'ja-JP-mac' if platform == 'osx' and locale == 'ja' else locale
         _platform = 'win' if platform == 'winsha1' else platform
 
+        # Bug 1340087 - Only include funnelcake params for Windows en-US builds.
+        _include_funnelcake_param = funnelcake_id and _platform == 'win' and _locale == 'en-US'
```

Comment:
I think for ease of upading this should we be wrong or some funnelcake someday expand compat that we should have settinngs for which platforms and locales are funnelcake enabled. Something like `FUNNELCAKE_PLATFORMS` and `FUNNELCAKE_LOCALES` that are pulled from the environment using the CSV parser and with these defaults. It might be overkill and shouldn't block this, but it would make us not have to push to prod if we're wrong about any of this.